builtin/providers/google/resource_compute_address.go

```diff
@@ -14,7 +14,12 @@ func resourceComputeAddress() *schema.Resource {
 		Create: resourceComputeAddressCreate,
 		Read:   resourceComputeAddressRead,
 		Delete: resourceComputeAddressDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Google's API doesn't provide a "name" field, so the custom importer is copying the id over to name. I was thinking of making a `IdToNameImporter` that could be used generically as I saw this done in one other place but I wasn't sure where to place that.